packages/es-dev-server/README.md

```diff
@@ -218,20 +137,22 @@ If you run the `es-dev-server` command from the root of the project, you can acc
 
 </details>
 
-### index.html in a Subfolder
+### index.html in a folder
 
-If you move your `index.html` inside a subfolder:
+If you move your `index.html` outside the root of your project, you have some different options.
 
 <details>
-  <summary>Use the `--open` parameter for when you'd like to keep you index.html in a subfolder.</summary>
+  <summary>Read more</summary>
+
+Use the `--open` parameter for when you'd like to keep you index.html in a subfolder.
 
 ```
 node_modules/...
 src/...
 src/index.html
 ```
 
-You can access your app in the browser at `/src/` or `/src/index.html`. You can tell `es-dev-server` to explicitly open at this path:
+You can access your app in the browser at `/src/` or `/src/index.html`. You can tell EDS to explicitly open at this path:
```

Comment:
hmmm I'm not sure if we want to start introducing acronyms in documentations? `EDS` is good for internal discussions but can be confusing if it's a first time reader who potentially only skims over the docs to find a specific option

Reply:
Fair enough, changed it back.